messenger/messenger_logic.go

```diff
@@ -73,13 +73,13 @@ func (a *App) postMessage(w http.ResponseWriter, r *http.Request) {
 	}
 
 	// send transaction to the sequencer
-	resp, err := a.sequencerClient.SendMessage(txEncoded)
+	err = a.sequencerClient.SendMessageViaComposer(txEncoded)
 	if err != nil {
 		log.Errorf("error sending message: %s\n", err)
 		w.WriteHeader(http.StatusInternalServerError)
 		return
 	}
-	log.WithField("responseCode", resp.Code).Debug("transaction submission result")
+	log.Infof("succesfully sent transaction to composer")
```

Comment:
please keep the structured logging to log the response code in debug mode

Reply:
problem is that, gRPC collector returns an empty response. so there is nothing to log really.